### PR TITLE
465 hosting tabs

### LINKED ISSUE #465: Hosting provider search only searches the visible providers
It doesn't search across all the categories, only the one currently showing.

### PR DESCRIPTION
**Pull request by lschatzkin**
closes #465 #472
Adds tab styling, all hosts tabs

### CODE REVIEW

**Comment by lschatzkin** [comment]
Not sure why build is still failing.

**Comment by bmw** [comment]
I hate git submodules...

The `_docs` submodule in this branch is on commit 3077b5150038ecaa8502cf586052f37cdf2dbad9 while the `master` branch in this repo is on b45f79d0ab5b8f0a84580880e8db9f28ae6b1f80.

If you update the submodule in this PR to match `master`, the build failures should go away.

**Comment by lschatzkin** [comment]
I've never used git submodule before, maybe it's for the best? I updated it in my sandbox but then there was nothing to push so... not sure what to do next.

File: _includes/hosting_providers_table.html [hunk 1]
```diff
@@ -13,7 +17,15 @@
     <div id="no-https" class="hosting-tab">
       <img alt="Red X icon" src="/images/NoHTTPSSupport.svg">
       <br class="mobile-only">
-      no HTTPS support
+      no HTTP support
```
Comment (wioux) [inline _includes/hosting_providers_table.html]: Looks like a typo here, it should be `no HTTPS support`.

**Comment by wioux** [comment]
Looks good! It'd be nice if this new tab were the default. It looks like that's controlled by the last line in main.js.

**Comment by lschatzkin** [comment]
> Looks good! It'd be nice if this new tab were the default. It looks like that's controlled by the last line in main.js.

Which tab is the default is a design decision, so let's ask @okaysoraya .

**Comment by lschatzkin** [comment]
> > Looks good! It'd be nice if this new tab were the default. It looks like that's controlled by the last line in main.js.
> 
> Which tab is the default is a design decision, so let's ask @okaysoraya .

I'm thinking I prefer the ALL tab to be the default, what are your thoughts on defaulting to the NO tab?

**Pull request merged**